src/Tools/IdeCoreBenchmarks/IncrementalSourceGeneratorBenchmarks.cs

```diff
@@ -97,6 +97,9 @@ private Task LoadSolutionAsync()
 
             var solution = _workspace.OpenSolutionAsync(_solutionPath, progress: null, CancellationToken.None).Result;
             Console.WriteLine("Finished opening roslyn: " + (DateTime.Now - start));
+
+            //foreach (var diag in _workspace.Diagnostics)
+            //    Console.WriteLine(diag);
```

Comment:
this is all personal code to let me hack away at things and get easily comparable results when i make changes.  not for review :)